docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
+services:
+  bot:
+    build: .
+    depends_on: ["redis"]
+  redis:
+    image: "redis:alpine"
```

Comment:
I wasn't sure whether or not to go ahead with including redis, so I just did it anyway.